add_to_app/android_books/.gitignore

```diff
@@ -0,0 +1,14 @@
+*.iml
```

Comment:
@abd99 I must have missed when we wound up with updated add_to_app/android_books in the testing branch, can you please revert?